.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+
+jdk:
+  - oraclejdk8
+cache:
```

Comment:
Make sure to delete `.m2/repository/com/teradata` in before_cache section - else the build results can be affected by previous builds.